gatherers/manifest.js

```diff
@@ -71,7 +72,11 @@ class Manifest extends Gather {
           })
           .then(manifestURL => Manifest._loadFromURL(options, manifestURL));
         })
-        .then(manifest => ({manifest}));
+        .then(manifestContent => {
+          return {
+            manifest: manifestParser(manifestContent).value
```

Comment:
yeah, I was just writing the same, since we don't distinguish between missing/erroneous values at this point